modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemIcon from '@elementor/ui/ListItemIcon';
+import ListItemText from '@elementor/ui/ListItemText';
+import Collapse from '@elementor/ui/Collapse';
+import ChevronDownIcon from '@elementor/icons/ChevronDownIcon';
+import RadioButtonUncheckedIcon from '@elementor/icons/RadioButtonUncheckedIcon';
+import CardMedia from '@elementor/ui/CardMedia';
+import Link from '@elementor/ui/Link';
+
+function CheckListItem( props ) {
+	const { id, title, description, link, CTA } = props.step,
+		{ expandedIndex, setExpandedIndex } = props,
+		[ expanded, setExpanded ] = useState( false );
```

Comment:
You don't need a state for this, when the props change it will re render (and you are passing the setter so they will change), just pass the item's index
```suggestion
		expanded = props.index === expandedIndex;
```

Reply:
it seems you do need it, for close and open to work, without it only open behavior is triggered